### ISSUE TITLE
Make mobile drawer size adjustable

### ISSUE THREAD

**Issue by amot-dev** [amot-dev/mototwist#99]
Should be able to drag it up to almost the full page and drag it down to almost nothing (just a handle). Would make it a lot easier to navigate the mobile site.

**Comment by amot-dev** [comment]
No longer as big of a deal with #115 being done.
